.envrc

```diff
@@ -1 +1,4 @@
+if ! has nix_direnv_version || ! nix_direnv_version 3.0.4; then
```

Comment:
Please add a comment explaining why is this needed.

Reply:
Replit needs that check for direnv to get picked up when you reconnect, if it's not there when you reconnect in browser then it kicks you out of the nix develop shell and you have to run `nix develop` again, not a big deal if I remove

Reply:
I think just adding this comment in the code would go a long way re maintainability.

Reply:
Yeah, I don't think it hurts to have it here, but we just need a comment why it is there, otherwise it looks odd, unnecessary and even suspicious.